### ISSUE TITLE
Integrating and building RayGun4Apple in macOS Sequoia with Xcode 16 fails

### ISSUE THREAD

**Issue by testsys-varadhu** [MindscapeHQ/raygun4apple#59]
Hi,

I am integrating RayGun4Apple with my application as a pod. After updating to macOS Sequoia and Xcode 16, I am encountering the following errors in RayGun4Apple. We would really appreciate a quick workaround or fix at the earliest. Thanks.

```
%My Project Path%/Pods/raygun4apple/Sources/Raygun_KSCrash/Recording/Monitors/Raygun_KSCrashMonitor_CPPException.cpp:62:13: No type named 'terminate_handler' in namespace 'std'

%My Project Path%/Pods/raygun4apple/Sources/Raygun_KSCrash/Recording/Monitors/Raygun_KSCrashMonitor_CPPException.cpp:207:47: No member named 'set_terminate' in namespace 'std'

%My Project Path%/Pods/raygun4apple/Sources/Raygun_KSCrash/Recording/Monitors/Raygun_KSCrashMonitor_CPPException.cpp:211:18: No type named 'set_terminate' in namespace 'std'
```

**Comment by sumitramanga** [comment]
Hi @testsys-varadhu 

Thank you for raising this issue!

We're currently working on a fix for this issue. You can refer to this thread here - https://github.com/MindscapeHQ/raygun4apple/issues/57

We'll update everyone once the fix is in production.

Thank you for your patience whilst we get his out!

Kind regards,
Sumitra

**Comment by testsys-varadhu** [comment]
Adding @Murali ***@***.***> & @Supriya ***@***.***>.

Varadharajan Venkat
Pronouns: he/him/his
Team Lead
[wAAAABJRU5ErkJggg==]<https://www.testsys.com/>
T: +91 960 072 8166
E: ***@***.******@***.***>
www.testsys.com<https://testsys.com/>
[tvx359lBdBeAgZ+9v8CDAB+j9k+Z4UpJQAAAABJRU5ErkJggg==]<https://www.linkedin.com/company/internettestingsystems> [wIMANrYSARYfBQ4AAAAAElFTkSuQmCC] <https://youtube.com/playlist?list=PL-9jlOkro7ls5JNyeSSalVHBJOt79HMUn>  [DEGNAAAAAElFTkSuQmCC] <https://testsys.com/webinars>


From: Sumitra Manga ***@***.***>
Date: Friday, September 20, 2024 at 10:25 AM
To: MindscapeHQ/raygun4apple ***@***.***>
Cc: Varadharajan Venkat ***@***.***>, Mention ***@***.***>
Subject: Re: [MindscapeHQ/raygun4apple] Integrating and building RayGun4Apple in macOS Sequoia with Xcode 16 fails (Issue #59)
Warning: External message from a sender outside of ITS.
Do not click links or open attachments unless you verify it is safe.
Do not provide your username, password, or any other personal information.


Hi @testsys-varadhu<https://github.com/testsys-varadhu>

Thank you for raising this issue!

We're currently working on a fix for this issue. You can refer to this thread here - #57<https://github.com/MindscapeHQ/raygun4apple/issues/57>

We'll update everyone once the fix is in production.

Thank you for your patience whilst we get this out!

Kind regards,
Sumitra

**Comment by Pallesen01** [comment]
@testsys-varadhu updating to the just released [raygun4apple 2.1.2](https://github.com/MindscapeHQ/raygun4apple/releases/tag/2.1.2) should solve your issue. Thanks for raising this 😃
